helms/legion/templates/edi/server.yaml

```diff
@@ -21,49 +21,18 @@ spec:
       {{- if .Values.rbac}}
       serviceAccountName: "{{ .Release.Name }}-edi"
       {{- end}}
-      volumes:
-      - name: cluster-secrets
-        secret:
-          secretName: "{{ .Release.Name }}-cluster-secrets"
-      {{- if and .Values.modelApiSecurity.enabled (eq .Values.modelApiSecurity.integration "jwt") }}
-      - name: jwt-config
-        secret:
-          secretName: "{{ .Release.Name }}-jwt-config"
-      {{- end}}
       containers:
       - name: edi
         image: "{{ include "legion.image-name" (dict "root" . "service" .Values.edi "tpl" "%sk8s-edi:%s") }}"
         resources:
           {{- toYaml .Values.edi.resources | nindent 10 }}
         env:
-        - name: DEPLOYMENT
-          value: "{{ .Release.Name }}"
         - name: NAMESPACE
           value: "{{ .Release.Namespace }}"
         - name: LEGION_API_ADDR
           value: "0.0.0.0"
         - name: LEGION_API_PORT
           value: "5000"
-        - name: VERBOSE
-          value: "1"
-        - name: MODEL_INSTANCE_SERVICE_ACCOUNT_NAME
-          value: "{{ .Release.Name }}-model"
-        {{- if .Values.rbac}}
-        - name: DOCKER_REGISTRY
-          value: "{{ .Values.modelStorage.external.url }}"
-        - name: DOCKER_REGISTRY_PROTOCOL
-          value: "{{ .Values.modelStorage.external.protocol }}"
-        - name: DOCKER_REGISTRY_USER
```

Comment:
Where this env variables are now?

Reply:
The variables are in the operator statefulset.
EDI server don't need these variables any more.